stable/prometheus/templates/alertmanager-serviceaccount.yaml

```diff
@@ -1,3 +1,4 @@
+{{- if .Values.alertmanager.enabled }}
```

Comment:
please also add this conditional [here](https://github.com/indigoid/charts/blob/831c9fcb5c500590dc0396a44b96ba2b222b97ee/stable/prometheus/templates/alertmanager-networkpolicy.yaml)